action-scripts/brownie/scripts/report_gauges.py

```diff
@@ -106,6 +108,14 @@ def _parse_added_transaction(transaction: dict, **kwargs) -> Optional[dict]:
     if not gauge_type:
         print("No gauge type found! Cannot process transaction")
         return
+    if transaction['to'] != ADDR_BOOK.search_unique("20230519-gauge-adder-v4").address:
```

Comment:
What's the point of this change?

Reply:
a deployment named ...migration-v3-v4/GaugeAdder popped up and was causing search unique to fail.  I made this more specific though to keep the specific contract inclusion.  It worked without it in this case because the deployment only has 1 contract.